pkg/neonvm/controllers/vmmigration_controller.go

```diff
@@ -713,6 +713,7 @@ func (r *VirtualMachineMigrationReconciler) targetPodForVirtualMachine(
 	pod.Name = migration.Status.TargetPodName
 
 	// add env variable to turn on migration receiver
+	// TODO: make it false or empty after the migration is done to enable correct readiness probe
```

Comment:
question(non-blocking): what's the plan here? I don't think we can modify container environments at runtime... And, is this TODO tracked somewhere?

Reply:
I think we discussed it verbally with @Omrigan and I promised to create an issue for that,  since I am a bit afraid of the scenario like:
- we start VM
- we migrate VM to a new pod
- new pod has `RECEIVE_MIGRATION` eq true which means that in some hypothetical scenario of restarting pod (which tbh I am not even sure is possible in our lifecycle) qemu will not boot, but will wait for some migration (which is not actually happening at the moment). 
Since we have already an issue to audit vmmigration controller, I will add it as a context there.